### PR TITLE
feat(ci): Add action for bumb version

### PR DESCRIPTION
**Pull request by linuxluigi**
Add action for bumb version

Fix https://github.com/SchwarzIT/go-template/issues/180

### CODE REVIEW

**Comment by linuxluigi** [comment]
Tested on fork and works fine 🙂

File: .github/workflows/bumb-version.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
```
Comment (brumhard) [inline .github/workflows/bumb-version.yml:1]: Could we somehow reuse the `hack/release.sh` script here?

File: .github/workflows/tag_release.yaml [hunk 2]
```diff
@@ -33,3 +36,5 @@ jobs:
         run: |
           git tag ${{ steps.tag.outputs.tag }}
           git push --tags
+      - name: Create release
+        run: gh release create {{ steps.tag.outputs.tag }} --verify-tag --generate-notes
```
Comment (brumhard) [inline .github/workflows/tag_release.yaml:40]: I don't really understand this change. The workflow should just create the tag and let the release workflow (with goreleaser) take care of creating the release with proper assets.

File: Makefile [hunk 3]
```diff
@@ -93,6 +93,10 @@ testing-project-ci:  ## Creates for all yml files in ./test_project_values a tes
 		make testing-project-ci-single VALUES_FILE=$$VALUES; \
 	done
 
+svu: ## Creates a new version, args options major, minor & patch
+	@go run github.com/caarlos0/svu $(args) --strip-prefix > config/version.txt
+	@go run github.com/caarlos0/svu $(args)
+
```
Comment (brumhard) [inline Makefile:99]: We already have `make release`. Why is that needed?

**Review by brumhard — CHANGES_REQUESTED** [review]

File: .github/workflows/bumb-version.yml (hunk 1 above)
Comment (linuxluigi) [reply .github/workflows/bumb-version.yml:1]: Oh, thanks for the reminder, I forget to take a look into it. I was mostly gone through a tutorial.

File: Makefile (hunk 3 above)
Comment (linuxluigi) [reply Makefile:99]: To bumb version in worklfow based on major, minor & patch. With this is possible to bumb version from the Github Workflow  UI.

Screenshot 2023-02-26 at 14 34 55
